src/Common/src/System/IO/DelegatingStream.cs

```diff
@@ -172,6 +172,11 @@ public override void EndWrite(IAsyncResult asyncResult)
             _innerStream.EndWrite(asyncResult);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
+        {
+            _innerStream.CopyTo(destination, bufferSize);
+        }
```

Comment:
> the way you get such a stream is via async APIs: you're less likely at that point to switch to using sync ones

Really good point. Also, since this is in Common, if I move the override into a separate file people might forget to include it. I undid the changes here since it doesn't seem to be worth the trouble.